BrowserKit/Sources/ContentBlockingGenerator/JsonHelper.swift

```diff
@@ -2,28 +2,49 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/
 
+import Common
 import Foundation
 
 struct JsonHelper {
+    private let logger: Logger
+
+    init(logger: Logger = DefaultLogger.shared) {
+        self.logger = logger
+    }
+
     func jsonEntityListFrom(filename: String) -> [String: Any] {
         let file = URL(fileURLWithPath: filename)
 
         do {
             let data = try Data(contentsOf: file)
-            return try JSONSerialization.jsonObject(with: data, options: []) as! [String: Any]
+            if let jsonObject = try JSONSerialization.jsonObject(with: data, options: []) as? [String: Any] {
+                return jsonObject
+            } else {
+                logger.log("Invalid JSON format for parsing entity list as dictionary from the file \(filename)",
```

Comment:
I'd remove also here the logging since this files are used inside the ExcecutableContentGenerator swift package, so a fatal error is enough here.